### PR TITLE
Run experimental tests against docker 1.9.1 and kubernetes 1.3 by using

### PR DESCRIPTION
**Pull request by dchen1107**
container-vm-v20160321 image.

cc/ @zmerlynn @ixdy

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml [hunk 1]
```diff
@@ -479,3 +479,29 @@
...
+    trigger-job: 'kubernetes-build'
+    test-owner: 'saadali@google.com'
+    provider-env: ''
+    emails: 'saadali@google.com,dawnchen@google.com'
+    suffix:
+        - 'gce-pd-test':
+            description: 'Run E2E slow tests on GCE test endpoint.'
+            timeout: 150
+            job-env: |
+                export GINKGO_TEST_ARGS="--ginkgo.focus=\[Slow\] \
```
Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: You can focus on just the PD tests here, right?

Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: Yeah, looking at this, I think you can just make a minor change to "--ginkgo.focus=Disks" and it should get what you need? (Trying to get my project in a state where I can run a dryrun.)

Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: Yeah, `=Disks` captures what you need:
```
$ hack/ginkgo-e2e.sh -ginkgo.dryRun=true --ginkgo.focus=Disks
Setting up for KUBERNETES_PROVIDER="gce".
Your active configuration is: [default]

Project: artful-fortress-786
Zone: us-central1-b
Jun 27 13:15:12.631: INFO: Fetching cloud provider for "gce"

I0627 13:15:12.631870   32168 gce.go:276] Using DefaultTokenSource &oauth2.reuseTokenSource{new:(*oauth2.tokenRefresher)(0xc2082d13b0), mu:sync.Mutex{state:0, sema:0x0}, t:(*oauth2.Token)(0xc208385bc0)}
I0627 13:15:13.232649   32168 e2e.go:276] Starting e2e run "da9243a6-3ca3-11e6-8d45-2c44fd1518ec" on Ginkgo node 1
Running Suite: Kubernetes e2e suite
===================================
Random Seed: 1467058512 - Will randomize all specs
Will run 4 of 274 specs

SSSSSSSSSSSSSSS
------------------------------
Pod Disks 
  should schedule a pod w/ a readonly PD on two hosts, then remove both. [Slow]
  /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/pd.go:168
•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
------------------------------
Pod Disks 
  should schedule a pod w/two RW PDs both mounted to one container, write to PD, verify contents, delete pod, recreate pod, verify contents, and repeat in rapid succession [Slow]
  /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/pd.go:270
•SSSSSSSSSSSSS
------------------------------
Pod Disks 
  should schedule a pod w/ a RW PD, remove it, then schedule it on another host [Slow]
  /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/pd.go:121
•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
------------------------------
Pod Disks 
  should schedule a pod w/ a RW PD shared between multiple containers, write to PD, delete pod, verify contents, and repeat in rapid succession [Slow]
  /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/pd.go:216
•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
Ran 4 of 274 Specs in 0.007 seconds
SUCCESS! -- 0 Passed | 0 Failed | 0 Pending | 270 Skipped PASS

Ginkgo ran 1 suite in 710.780141ms
Test Suite Passed
```

Comment (dchen1107) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: Chat with Saadi, and he suggested focus on PD only.

**Comment by dchen1107** [comment]
cc/ @saad-ali @matchstick

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml [hunk 2]
```diff
@@ -479,3 +479,29 @@
...
+    trigger-job: 'kubernetes-build'
+    test-owner: 'saadali@google.com'
+    provider-env: ''
+    emails: 'saadali@google.com,dawnchen@google.com'
+    suffix:
+        - 'gce-pd-test':
+            description: 'Run E2E slow tests on GCE test endpoint.'
+            timeout: 150
+            job-env: |
+                export GINKGO_TEST_ARGS="--ginkgo.focus=RW\sPD,\sremove\sit,\sthen\sschedule\sit\son\sanother\shost \
```
Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: This is capturing way more, about 79 specs.

Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: Oh, nevermind. Bug in how I was testing. However, please take out the `--ginkgo.skip` section below.

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml [hunk 3]
```diff
@@ -479,3 +479,29 @@
...
+                                         --ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
+                export PROJECT="kubekins-e2e-gce-pd-test"
```
Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: Does this project exist?

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml [hunk 4]
```diff
@@ -479,3 +479,29 @@
...
+                export CLOUDSDK_BUCKET="gs://cloud-sdk-testing/rc"
+                export JENKINS_USE_SERVER_VERSION="y"
+                export E2E_NAME="gce-pd-e2e-test"
+                export KUBE_OS_DISTRIBUTION="debian"
+                export KUBE_GCE_MASTER_IMAGE="container-v1-3-v20160604"
```
Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: Are you keeping the master at Docker 1.11.2 intentionally?

Comment (dchen1107) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: There is no issue on master side. Don't want to change at this moment.

Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: SGTM

**Comment by zmerlynn** [comment]
Squash and I'll merge.

**Comment by dchen1107** [comment]
Done squash. Thanks!

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml [hunk 5]
```diff
@@ -479,3 +479,28 @@
...
         - 'kubernetes-e2e-{suffix}'
+
+# Jobs that run e2e tests on GCE with container-vm-v20160321 image to do some
+# experimental (docker 1.9.1 and Kubernetes 1.3) for
+# https://github.com/kubernetes/kubernetes/issues/27691
+- project:
+    name: kubernetes-e2e-gce-pd
+    trigger-job: 'kubernetes-build'
+    test-owner: 'saadali@google.com'
+    provider-env: ''
```
Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: Why was this nulled out? I missed this the first time around.

Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: #227

**Pull request merged**